### PR TITLE
Add reward_from_db cli command

### PR DESCRIPTION
**Pull request by maplant**

### CODE REVIEW

**Review by bbalser — APPROVED** [review]

File: poc_mobile_verifier/src/cli/reward_from_db.rs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    start: NaiveDateTime,
+    #[clap(long)]
+    end: NaiveDateTime,
+}
+
+impl Cmd {
+    pub async fn run(self) -> Result {
+        let Self { start, end } = self;
+
+        let start = DateTime::from_utc(start, Utc);
```
Comment (jeffgrunewald) [inline poc_mobile_verifier/src/cli/reward_from_db.rs:26]: any reason to pattern match the start and end out above vs. `self.start` and `self.end` here or just preference?

File: poc_mobile_verifier/src/cli/reward_from_db.rs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            .rewards
+            .iter()
+            .map(|r| (PublicKey::try_from(r.account.as_slice())?, r.amount))
+            .collect();
+
+        println!(
+            "{}",
+            serde_json::to_string_pretty(&json!({
+                "rewards": rewards,
+                "total_rewards": total_rewards,
```
Comment (jeffgrunewald) [inline poc_mobile_verifier/src/cli/reward_from_db.rs:64]: can we have a CYA here like `expected_total` or similar that computes the number of mobile bones we expect to emit based on the duration of the epoch passed in on the command? then you can very quickly see on the command output that you're awarding the amount you expect

**Review by jeffgrunewald — APPROVED** [review]
one nice-to-have; otherwise lgtm

File: poc_mobile_verifier/src/cli/reward_from_db.rs (hunk 1 above)
Comment (maplant) [reply poc_mobile_verifier/src/cli/reward_from_db.rs:26]: just preference

File: poc_mobile_verifier/src/cli/reward_from_db.rs (hunk 2 above)
Comment (maplant) [reply poc_mobile_verifier/src/cli/reward_from_db.rs:69]: sure

**Review by jeffgrunewald — APPROVED** [review]
🚀

**Pull request merged**